src/internal/m365/service/groups/export.go

```diff
@@ -20,6 +25,7 @@ func ProduceExportCollections(
 	exportCfg control.ExportConfig,
 	opts control.Options,
 	dcs []data.RestoreCollection,
+	backupDriveIDNames idname.CacheBuilder,
```

Comment:
Write methods aren't required here.
```suggestion
	backupDriveIDNames idname.Cache,
```